libs/librepcb/common/graphics/graphicslayer.cpp

```diff
@@ -232,54 +232,82 @@ void GraphicsLayer::getDefaultValues(const QString& name, QString& nameTr,
     h.insert(sSymbolPinNames,           {tr("Pin Names"),             QColor(64, 64, 64, 255),    Qt::gray,                   true});
     h.insert(sSymbolPinNumbers,         {tr("Pin Numbers"),           QColor(64, 64, 64, 255),    Qt::gray,                   true});
     // board asymmetric
-    h.insert(sBoardSheetFrames,         {tr("Sheet Frames"),          Qt::lightGray,              Qt::white,                  true});
-    h.insert(sBoardOutlines,            {tr("Board Outlines"),        QColor(255, 255, 255, 180), QColor(255, 255, 255, 220), true});
-    h.insert(sBoardMillingPth,          {tr("Milling (PTH)"),         QColor(0, 255, 255, 150),   QColor(0, 255, 255, 220),   true});
-    h.insert(sBoardDrillsNpth,          {tr("Drills (NPTH)"),         QColor(255, 255, 255, 200), QColor(255, 255, 255, 255), true});
-    h.insert(sBoardPadsTht,             {tr("Pads"),                  QColor(0, 255, 0, 150),     QColor(0, 255, 0, 220),     true});
-    h.insert(sBoardViasTht,             {tr("Vias"),                  QColor(0, 255, 0, 150),     QColor(0, 255, 0, 220),     true});
+    h.insert(sBoardSheetFrames,         {tr("Sheet Frames"),          QColor("#96E0E0E0"),        QColor("#FFFFFFFF"),        true});
+    h.insert(sBoardOutlines,            {tr("Board Outlines"),        QColor("#C8FFFFFF"),        QColor("#FFFFFFFF"),        true});
+    h.insert(sBoardMillingPth,          {tr("Milling (PTH)"),         QColor("#C8FFFFFF"),        QColor("#FFFFFFFF"),        true});
```

Comment:
I would suggest ![00DDFF](https://placehold.it/15/00DDFF/000000?text=+) and ![00FFFF](https://placehold.it/15/00FFFF/000000?text=+) to make it distinguishable from board outlines:

```suggestion
    h.insert(sBoardMillingPth,          {tr("Milling (PTH)"),         QColor("#C800DDFF"),        QColor("#FF00FFFF"),        true});
```

Reply:
Looks good!